src/handlers/read-storage-handler.js

```diff
@@ -24,7 +24,7 @@ const _noOp = () => undefined
 export function StorageHandler (storageStrategy, externalStorageHandler) {
   const errors = []
   function _externalOrError (functionName) {
-    const hasExternal = externalStorageHandler && externalStorageHandler[functionName] && isFunction(externalStorageHandler[functionName])
```

Comment:
I would move the `!strEqualsIgnoreCase(storageStrategy, StorageStrategy.disabled)` to the beginning of the and chain as it's a bit more explicit that we really don't care about the externalStorageHandler in that case